ui/src/app/pages/runtimes-list.tsx

```diff
@@ -10,6 +10,23 @@ import { NavigationProps } from 'app/utils/navigation';
 import { withNavigation } from 'app/utils/with-navigation-hoc';
 import { ajaxContext, Environments } from 'terraui/out/Environments';
 
+// Hidden columns Index
+const workspace = 2;
+const delete_cloud_environment = 11;
+const status_pd = 5;
+const delete_pd = 10;
```

Comment:
I like your instinct to make these consistent. When namespacing, put the category first so sorted lists automatically group by category, e.g.,
```suggestion
const pdStatus = 5;
const pdDelete = 10;
```